code/modules/antagonists/bloodsucker/powers/brawn.dm

```diff
@@ -107,7 +107,7 @@
 	if(user_C.handcuffed)
 		var/obj/O = user_C.get_item_by_slot(SLOT_HANDCUFFED)
 		if(istype(O))
-			//user_C.visible_message("<span class='warning'>[user_C] attempts to remove [O]!</span>", \
```

Comment:
Just delete the bloodsucker power comments out when they are causing issues, no point in keeping them in, the old dev that worked on it REALLY liked their in depths comments